x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -189,26 +217,28 @@ export const MisconfigurationFindingsDetailsTable = memo(
         ),
       },
       {
-        field: 'result',
-        render: (result: CspFindingResult) => <CspEvaluationBadge type={result?.evaluation} />,
+        field: MISCONFIGURATION.RESULT_EVALUATION,
+        render: (result: 'failed' | 'passed' | undefined) => <CspEvaluationBadge type={result} />,
         name: i18n.translate(
           'xpack.securitySolution.flyout.left.insights.misconfigurations.table.resultColumnName',
           {
             defaultMessage: 'Result',
           }
         ),
         width: `${resultWidth}px`,
+        sortable: true,
       },
       {
-        field: 'rule',
-        render: (rule: CspBenchmarkRuleMetadata) => <EuiText size="s">{rule?.name}</EuiText>,
```

Comment:
same here, its a nit pick in this case but usually doing something like `CspBenchmarkRuleMetadata['ruleName']` would be a more scaleable approach